components/kyma-environment-broker/internal/servicemanager/metadata.go

```diff
@@ -0,0 +1,42 @@
+package servicemanager
+
+import (
+	"fmt"
+
+	kebError "github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/error"
+)
+
+type Metadata struct {
+	ServiceID string
+	BrokerID  string
+	PlanID    string
+}
+
+func GenerateMetadata(c Client, offeringName, planName string) (*Metadata, error) {
```

Comment:
'Metadata' does not say anything important, let's change the name and use "Offering" in that name